src/util-exception-policy.c

```diff
@@ -88,7 +88,10 @@ enum ExceptionPolicy ExceptionPolicyParse(const char *option, const bool support
             policy = EXCEPTION_POLICY_IGNORE;
             SCLogConfig("%s: %s", option, value_str);
         } else {
-            SCLogConfig("%s: ignore", option);
+            FatalErrorOnInit(SC_ERR_INVALID_ARGUMENT,
+                    "\"%s\" is not a valid exception policy value,  please check Suricata "
```

Comment:
instead of, or in addition to, suggesting to RTFM, should we list the valid options here? `"Valid options are drop-flow, pass-flow, ... "`.

Reply:
Yes, yes, yes, my bad 😅  didn't mean to be rude to the users XD